proto/panacea/datapool/v2/genesis.proto

```diff
@@ -6,4 +6,6 @@ option go_package = "github.com/medibloc/panacea-core/x/datapool/types";
 import "google/protobuf/any.proto";
 
 // GenesisState defines the datapool module's genesis state.
-message GenesisState {}
\ No newline at end of file
+message GenesisState {
+  uint64 next_pool_number = 1;
```

Comment:
In next PRs, we need to add `pools` here. Please add a TODO comment for now.

Reply:
Also added TODO comment in genesis.proto. Thanks!